docs/content/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -211,4 +223,41 @@ Flink currently only provides processing guarantees for jobs without iterations.
 
 Please note that records in flight in the loop edges (and the state changes associated with them) will be lost during failure.
 
+## Checkpointing with parts of the graph finished *(BETA)*
+
+Starting from Flink 1.14 it is possible to continue performing checkpoints even if parts of the job graph have finished processing all data, because it was a bounded source. The feature must be enabled
```

Comment:
First of all, I agree, this sentence is not very clear. I would not use `all` / `only`, because this way the reader might be confused that it works iff the job graph contains exclusively bounded sources.

I will rephrase it to: 
> Starting from Flink 1.14 it is possible to continue performing checkpoints even if parts of the job
graph have finished processing all data, which might happen if it contains bounded sources. The
WDYT?